src/config.rs

```diff
@@ -1712,6 +1715,14 @@ impl UnifiedReadPoolConfig {
                     .into(),
             );
         }
+        let limit = SysQuota::cpu_cores_quota() as usize;
+        if self.max_thread_count > limit {
```

Comment:
The default `max-thread-count` is set to:
```rust
let cpu_num = SysQuota::cpu_cores_quota();
let mut concurrency = (cpu_num * 0.8) as usize;
concurrency = cmp::max(UNIFIED_READPOOL_MIN_CONCURRENCY, concurrency);
```
so this check may not even pass for the default config.

Reply:
Fixed, thanks!